### PR TITLE
[MM-T658] /invite - current channel

### PR DESCRIPTION
**Pull request by clarmso**
#### Summary

This test tests the basic functionality of `/invite`.

#### Ticket Link

https://automation-test-cases.vercel.app/test/MM-T658

#### Related Pull Requests

- Has server changes N/A
- Has redux changes N/A
- Has mobile changes N/A
#### Screenshots

Screen Shot 2020-08-16 at 1 37 39 AM

### CODE REVIEW

File: e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js [hunk 1]
```diff
@@ -243,6 +256,28 @@ describe('I18456 Built-in slash commands: common', () => {
...
+        cy.uiWaitUntilMessagePostedIncludes(`@${userGroup[0].username} added to the channel by you`);
+
+        // * Cannot invite deactivated users to a channel
+        cy.postMessage(`/invite @${deactivatedUser.username}`);
+        cy.uiWaitUntilMessagePostedIncludes('We couldn\'t find the user. They may have been deactivated by the System Administrator.');
+
+        cy.apiLogin(userGroup[0]);
+        cy.visit(`${team1.name}/channels/town-square`);
+
+        // * Added user sees channel added to LHS, mention badge, system message "username added to the channel by username."
```
Comment (clarmso) [inline e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js:276]: I don't see the "mention badge". Could someone please point out if the badge supposed to be there?

File: e2e/cypress/support/api/user.js [hunk 2]
```diff
@@ -203,3 +203,16 @@ Cypress.Commands.add('apiGetUsersNotInTeam', ({teamId, page = 0, perPage = 60} =
         cy.wrap({users: response.body});
     });
 });
+
+Cypress.Commands.add('apiDeactivateUser', (userId) => {
```
Comment (clarmso) [inline e2e/cypress/support/api/user.js:207]: Please let me know if it's the correct place to add a command.

File: e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js [hunk 3]
```diff
@@ -243,6 +256,28 @@ describe('I18456 Built-in slash commands: common', () => {
...
+        cy.uiWaitUntilMessagePostedIncludes(`@${userGroup[0].username} added to the channel by you`);
+
+        // * Cannot invite deactivated users to a channel
+        cy.postMessage(`/invite @${deactivatedUser.username}`);
+        cy.uiWaitUntilMessagePostedIncludes('We couldn\'t find the user. They may have been deactivated by the System Administrator.');
+
+        cy.apiLogin(userGroup[0]);
+        cy.visit(`${team1.name}/channels/town-square`);
+
+        // * Added user sees channel added to LHS, system message "username added to the channel by username."
```
Comment (clarmso) [inline e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js:276]: I don't see the "mention badge".

File: e2e/cypress/support/api/user.js (hunk 2 above)
Comment (saturninoabril) [reply e2e/cypress/support/api/user.js:207]: That's perfect. Thanks for adding this command 💯

File: e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js (hunk 3 above)
Comment (saturninoabril) [reply e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js:276]: Yep, it should be present. You need to logout before logging in as a new user.

File: e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js [hunk 4]
```diff
@@ -243,6 +256,28 @@ describe('I18456 Built-in slash commands: common', () => {
                 });
         });
     });
+
+    it('MM-T658 /invite - current channel', () => {
+        loginAndVisitDefaultChannel(user1, `${team1.name}/channels/${testChannel.name}`);
+
+        // # Post `/invite @username` where username is a user who is not in the current channel
+        cy.postMessage(`/invite @${userGroup[0].username}`);
```
Comment (saturninoabril) [inline e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js:264]: nit: Please create variable for active user to invite so it's easier to read.
``const userToInvite = userGroup[0];``

File: e2e/cypress/support/api/user.js (hunk 2 above)
Comment (hahmadia) [reply e2e/cypress/support/api/user.js:207]: Agreed. Useful command 👍

**Review by srkgupta — APPROVED** [review]
Awesome @clarmso. The test looks great and is working fine. Approving the PR. Thanks for your contribution.

**Pull request merged**